CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,184 @@
+needed in order to get the most helpful answers. Refrain from pasting multiple lines of
+code directly into the Slack channels - instead use gist.github.com or another paste site
+to provide code snippets.
+
+## Helpful
+
+The Nautobot community is committed to being a welcoming environment for all users,
+regardless of skill level. We were all beginners once upon a time, and our community
+cannot grow without an environment where new users feel safe and comfortable asking questions.
+It can become frustrating to answer the same questions repeatedly; however, community
+members are expected to remain courteous and helpful to all users equally, regardless of
+skill or knowledge level. Avoid providing responses that prioritize snideness and snark over
+useful information. At the same time, everyone is expected to read the provided
+documentation thoroughly. We are happy to answer questions, provide strategic guidance,
+and suggest effective workflows, but we are not here to do your job for you.
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+All Nautobot events and participants therein are governed by this Code of Conduct and
+anti-harassment policy. We expect organizers to enforce these guidelines throughout all events,
+and we expect attendees, speakers, sponsors, and volunteers to help ensure a safe
+environment for our whole community. Specifically, this Code of Conduct covers
+participation in all Nautobot-related forums and mailing lists, code and documentation
+contributions, public Slack channels, private correspondence, and public meetings.
+
+Where our community participation happens in joint with another community or organization,
+for example the Network to Code Slack, vendor-hosted conferences, etc., we expect strict
+adherence with their community policies as well, as we are guests of their spaces.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+community@networktocode.com.
```

Comment:
Is this a live email address?

Reply:
Added into the `nautobot-ansible` repo: https://github.com/nautobot/nautobot-ansible/blob/develop/CODE_OF_CONDUCT#L114

I assumed the powers at be did their due-diligence to set this up but it's worth verifying.